ElectronicObserver/Window/Tools/ExpeditionRecordViewer/ExpeditionRecordViewer.xaml

```diff
@@ -0,0 +1,301 @@
+							</commontools:ImageTextControl>
+						</DataTemplate>
+					</DataGridTemplateColumn.CellTemplate>
+				</DataGridTemplateColumn>
+
+				<DataGridTemplateColumn Header="{Binding DataContext.ExpeditionRecordViewer.Baux, Source={StaticResource Proxy}}" SortMemberPath="Bauxite">
+					<DataGridTemplateColumn.CellTemplate>
+						<DataTemplate>
+							<commontools:ImageTextControl HorizontalAlignment="Center" Text="{Binding Bauxite}">
+								<commontools:ImageTextControl.Image>
+									<common:IconContentIcon Type="ResourceBauxite" />
+								</commontools:ImageTextControl.Image>
+							</commontools:ImageTextControl>
+						</DataTemplate>
+					</DataGridTemplateColumn.CellTemplate>
+				</DataGridTemplateColumn>
+
+				<DataGridTemplateColumn Header="{Binding DataContext.ExpeditionRecordViewer.Item1, Source={StaticResource Proxy}}" SortMemberPath="ItemOneCount">
+					<DataGridTemplateColumn.CellTemplate>
+						<DataTemplate>
+							<Border Background="Transparent" ToolTip="{Binding ItemOneName}">
+								<commontools:ImageTextControl HorizontalAlignment="Center" Text="{Binding ItemOneCount}">
+									<commontools:ImageTextControl.Style>
+										<Style TargetType="commontools:ImageTextControl">
+											<Setter Property="Visibility" Value="Visible" />
+											<Style.Triggers>
+												<DataTrigger Binding="{Binding ItemOneId}">
+													<DataTrigger.Value>
+														<electronicObserverTypes:UseItemId>
+															Unknown
+														</electronicObserverTypes:UseItemId>
+													</DataTrigger.Value>
+
+													<Setter Property="Visibility" Value="Collapsed" />
+												</DataTrigger>
+											</Style.Triggers>
+										</Style>
+									</commontools:ImageTextControl.Style>
+
+									<commontools:ImageTextControl.Image>
+										<common:UseItemIcon Type="{Binding ItemOneId}" />
+									</commontools:ImageTextControl.Image>
+								</commontools:ImageTextControl>
+							</Border>
+						</DataTemplate>
+					</DataGridTemplateColumn.CellTemplate>
+				</DataGridTemplateColumn>
+
+				<DataGridTemplateColumn Header="{Binding DataContext.ExpeditionRecordViewer.Item2, Source={StaticResource Proxy}}" SortMemberPath="ItemTwoCount">
+					<DataGridTemplateColumn.CellTemplate>
+						<DataTemplate>
+							<Border Background="Transparent" ToolTip="{Binding ItemTwoName}">
+								<commontools:ImageTextControl HorizontalAlignment="Center" Text="{Binding ItemTwoCount}">
+									<commontools:ImageTextControl.Style>
+										<Style TargetType="commontools:ImageTextControl">
+											<Setter Property="Visibility" Value="Visible" />
+											<Style.Triggers>
+												<DataTrigger Binding="{Binding ItemTwoId}">
+													<DataTrigger.Value>
+														<electronicObserverTypes:UseItemId>
+															Unknown
+														</electronicObserverTypes:UseItemId>
+													</DataTrigger.Value>
+
+													<Setter Property="Visibility" Value="Collapsed" />
```

Comment:
This column will never be displayed ?

Reply:
I removed it for better scrolling performance, and didn't really see much value in showing the fleet here. Though I made it visible again now cause the column selector was showing it anyway.